java/autoscaler/src/main/java/com/example/bigtable/autoscaler/Autoscaler.java

```diff
@@ -88,6 +88,11 @@
    */
   public static double CPU_PERCENT_TO_UPSCALE = .7;
 
+  /**
+   * Number of seconds to sleep after a scaling operation to wait for changes to take effect.
+   */
+  private static int SLEEP_DURATION_S = 600;
```

Comment:
I'm suggesting a longer sleep after a scaling operation to let the impact of that scaling operation take effect. During my tests, if I get 70% CPU load on a 3 node cluster, then scaled to 5 or 6 nodes, it was usually enough to drive down CPU below 50%, but only after a few minutes. The way the Java sample is currently written it will check every minute and scale to 3, then 6, then 9, then 12, etc. So I'm suggesting after you scale wait a bit longer (10 min was arbitrary but seemed reasonable) before checking again to see the scaling operation's impact.